bin/commands/runs.js

```diff
@@ -17,6 +17,9 @@ module.exports = function run(args) {
   return utils.validateBstackJson(bsConfigPath).then(function (bsConfig) {
     utils.setUsageReportingFlag(bsConfig, args.disableUsageReporting);
 
+    // setting defaultAuthHash to {} if not present and set via env variables or via args.
```

Comment:
We should call this in other places, such as stop, info as well to make sure they're consistent as well.

Reply:
Done.